lib/slack/web/docs/bak

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I think this file can be deleted?

Reply:
I think most of these files are redundant. This needs another pass because of all of the conflicts.